packages/vaadin-lumo-styles/src/components/button.css

```diff
@@ -0,0 +1,329 @@
+  background: var(--vaadin-button-tertiary-background, var(--_background));
+  min-width: 0;
+}
+
+:host([theme~='tertiary']) {
+  border: var(--vaadin-button-tertiary-border, none);
+  color: var(--vaadin-button-tertiary-text-color, var(--lumo-primary-text-color));
+  font-weight: var(--vaadin-button-tertiary-font-weight, 500);
+  padding: var(--vaadin-button-tertiary-padding, 0 calc(var(--_button-size) / 6));
+}
+
+:host([theme~='tertiary-inline'])::before {
+  display: none;
+}
+
+:host([theme~='tertiary-inline']) {
+  margin: 0;
+  height: auto;
+  padding: 0;
+  line-height: inherit;
+  font-size: inherit;
+}
+
+:host([theme~='tertiary-inline']) [part='label'] {
+  padding: 0;
+  overflow: visible;
+  line-height: inherit;
+}
+
+:host([theme~='primary']) {
+  background: var(--_lumo-button-primary-background);
+  border: var(--vaadin-button-primary-border, none);
+  color: var(--_lumo-button-primary-text-color);
+  font-weight: var(--vaadin-button-primary-font-weight, 600);
+  min-width: calc(var(--lumo-button-size) * 2.5);
+}
+
+:host([theme~='primary'])::before {
+  background-color: black;
+}
+
+@media (any-hover: hover) {
+  :host([theme~='primary']:not([disabled]):hover)::before {
+    opacity: 0.05;
+  }
+}
+
+:host([theme~='primary'][active])::before {
+  opacity: 0.1;
+}
+
+:host([theme~='primary'][active])::after {
+  opacity: 0.2;
+}
+
+/* Colors (success, warning, error, contrast) */
+
+:host([theme~='success']) {
+  color: var(--lumo-success-text-color);
+}
+
+:host([theme~='success'][theme~='primary']) {
+  background-color: var(--lumo-success-color);
+  color: var(--lumo-success-contrast-color);
+}
+
+:host([theme~='warning']) {
+  color: var(--lumo-warning-text-color);
+}
+
+:host([theme~='warning'][theme~='primary']) {
+  background-color: var(--lumo-warning-color);
+  color: var(--lumo-warning-contrast-color);
+}
+
+:host([theme~='error']) {
+  color: var(--lumo-error-text-color);
+}
+
+:host([theme~='error'][theme~='primary']) {
+  background-color: var(--lumo-error-color);
+  color: var(--lumo-error-contrast-color);
+}
+
+:host([theme~='contrast']) {
+  color: var(--lumo-contrast);
+}
+
+:host([theme~='contrast'][theme~='primary']) {
+  background-color: var(--lumo-contrast);
+  color: var(--lumo-base-color);
+}
+
+/* Disabled state. Keep selectors after other color variants. */
+
+:host([disabled]) {
+  color: var(--lumo-disabled-text-color);
+  pointer-events: var(--_vaadin-button-disabled-pointer-events, none);
+  cursor: not-allowed;
+}
+
+:host([theme~='primary'][disabled]) {
+  background-color: var(--lumo-contrast-30pct);
+  color: var(--lumo-base-color);
+}
+
+:host([theme~='primary'][disabled]) [part] {
+  opacity: 0.7;
+}
+
+/* Icons */
+
+[part] ::slotted(vaadin-icon) {
+  display: inline-block;
+  width: var(--lumo-icon-size-m);
+  height: var(--lumo-icon-size-m);
+}
+
+/* Vaadin icons are based on a 16x16 grid (unlike Lumo and Material icons with 24x24), so they look too big by default */
```

Comment:
Could remove Material mention from here.

Reply:
Removed this comment entirely.